component/gitlab-ci.jsonnet

```diff
@@ -96,6 +96,8 @@ local GitLabCI() = {
     },
     variables: cloud_specific_variables[params.provider].apply,
     script: [
+      'apk add --no-cache curl',
```

Comment:
Why not add `curl` to the image used by this step?

Reply:
Because we currently use the upstream GitLab Terraform image from https://gitlab.com/gitlab-org/terraform-images and I don't really want to maintain a downstream image based on that image just so we can add `curl` to it.

I guess we could open an issue on that repository asking to add `curl` to the image.